hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java

```diff
@@ -1550,6 +1569,7 @@ public void testVersions() throws Exception {
   }
 
   @Test
+  @SuppressWarnings("checkstyle:MethodLength")
```

Comment:
Maybe in the future we will also split these methods so let's do not suppress the warning?

Reply:
I tried to do so. But it seems to me that these tests are doing a sequence of operations which cannot be broken without doing a lot of redundant initialisation logic. I believe that the gain of making these tests smaller doesn't overcome the hit on the efficiency.

To be honest, checkstyle rule for maximizing the size of integration tests doesn't make sense to me.